serpentTools/parsers/base.py

```diff
@@ -119,7 +119,7 @@ class SparseReaderMixin(object):
     ``sparse`` passed to the constructor and if :term:`scipy` is
     installed.
 
-    Paramters
+    Parameters
     ---------
```

Comment:
I think we'll need an extra `-` to get the docstring to render properly after this change
```suggestion
    ----------
```

Reply:
Good point, thanks